environments/development.env.template

```diff
@@ -0,0 +1,4 @@
+OCAML_BENCH_DOCKER_CPU=1
+OCAML_BENCH_DB_PASSWORD=docker
+OCAML_BENCH_GRAPHQL_KEY=secret
+OCAML_BENCH_TARGET_ARCH=arm64
```

Comment:
```suggestion
OCAML_BENCH_TARGET_ARCH=amd64
```

Reply:
Good catch! I'm actually on arm, forgot to switch this back.